docs/content/node-operation/node-setup.mdx

```diff
@@ -16,9 +16,12 @@ The hardware your Node will need varies depending on the role your Node will pla
 | **Execution**    | 64 cores  | 800 GB |  9 TB  | n2-highmem-128 |
 | **Verification** |  2 cores  | 16 GB  | 200 GB | n2-highmem-2   |
 | **Access**       |  16 cores | 64 GB  | 900 GB | n2-standard-16 |
+| **Observer**     |  2 cores  | 4 GB   | 300 GB | n2-highmem-16  |
 
 _Note: The above numbers represent our current best estimate for the state of the network. These will be actively updated as we continue benchmarking the network's performance._
 
+> For the observer node setup, follow this [guide](/node-operation/observer-node).
```

Comment:
Rephrase to: "To run an Observer node, follow this [guide](/node-operation/observer-node)."